.github/workflows/pullrequestclose.yml

```diff
@@ -5,18 +5,14 @@ concurrency:
   cancel-in-progress: true
 
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
does this need to be target?

Reply:
That was the main impetus for this. The GCP_CREDENTIALS secret isn't available to a normal pull request. As this does NOT actually checkout code, it seemed safe to just switch it to _target.

Reply:
Ah. Gotcha.